style.css/thirdsectionStyle.css

```diff
@@ -0,0 +1,113 @@
+.third-section {
```

Comment:
At the end of the section, we need space. Check your implementation in the browser and compare it with the design.

Reply:
You need to add padding-bottom: 64px; still I cannot see the space.

Reply:
Done.